pkg/clusterregistry/options/options.go

```diff
@@ -20,59 +20,56 @@ package options
 import (
 	"time"
 
+	"k8s.io/apiserver/pkg/server"
 	genericoptions "k8s.io/apiserver/pkg/server/options"
 	"k8s.io/apiserver/pkg/storage/storagebackend"
 
 	"github.com/spf13/pflag"
 )
 
+type OptionsGetter interface {
+	GenericServerRunOptions() *genericoptions.ServerRunOptions
+	SecureServing() *genericoptions.SecureServingOptions
+	Validate() []error
+	Audit() *genericoptions.AuditOptions
+	Features() *genericoptions.FeatureOptions
+	ApplyAuthentication(*server.Config) error
```

Comment:
Can you order these so that similar methods are grouped? Perhaps the getters, then `Validate`, then `ApplyX`?

Reply:
Done.